knowledge_base/Testing-Environments.md

```diff
@@ -1,11 +1,11 @@
 # Testing Environments
 
-Currently we have 2 open testing environments in Heroku for testing any third party integrations requiring a direct connection, called "Frick" and "Frack" and located at https//commonwealth-frick.herokuapp.com and https//commonwealth-frack.herokuapp.com. Frick has access to chain events, which Frack does not. Frack has mainly been used for testing CDN caching (which can't be tested locally) and any sus migrations (that need to be tested without merging into master.)
+Currently we have 2 open testing environments in Heroku for testing any third party integrations requiring a direct connection, called "Frick" and "Frack" and located at `https://commonwealth-frick.herokuapp.com` and `https://commonwealth-frack.herokuapp.com`. Frick has access to chain events, which Frack does not. Frack has mainly been used for testing CDN caching (which can't be tested locally) and any sus migrations (that need to be tested without merging into master.)
```

Comment:
Now that CE 2.0 has been deployed all environments have access to chain-events (Frack included).